source/renderer/app/containers/staking/dialogs/DelegationSetupWizardDialogContainer.js

```diff
@@ -182,6 +183,7 @@ export default class DelegationSetupWizardDialogContainer extends Component<
       hwDeviceStatus,
       sendMoneyRequest,
       selectCoinsRequest,
+      _getHardwareWalletDeviceInfoByWalletId,
```

Comment:
I may be wrong, but I think `_` means it's a private method. If it's a public one it could be removed

Reply:
@daniloprates you are in right, my bad